ChecksFile.wl

```diff
@@ -38,9 +38,6 @@ outfolder = outpath<>filename;
 
 
 
-On[Assert]

```

Comment:
Why do we no longer have asserts turned on?

Reply:
Because all asserts that were in the checks file have been removed in favor of error messages or re-working the tests.  There are sstill asserts in the StabilityPackage

Reply:
I see. Thanks!